src/smoothscroll.js

```diff
@@ -245,13 +245,13 @@ function polyfill() {
     if (shouldBailOut(arguments[0]) === true) {
       original.scroll.call(
         w,
-        arguments[0].left !== undefined
+        arguments[0] !== null && arguments[0].left !== undefined
           ? arguments[0].left
-          : typeof arguments[0] !== 'object'
```

Comment:
I noticed that the browser implementation of `window.scrollTo` handles `null`, whereas this ternary excludes `null` and passes a default of `scrollX` or `pageXOffset`.